[backport to 3.5] bpo-29474: Improve documentation for weakref.WeakValueDictionary

Mariatta opened the pull request:
There were some grammatical errors in weakref.WeakValueDictionary
documentation.

Same as https://github.com/python/cpython/pull/10, backported to 3.5 branch

auvipy commented:
you should send your PR at official python hg repo

williambr commented:
I have bumped into this issue as well.

This change would be much appreciate on my end.

What can we do to move the ball forward here?

Thanks!

brettcannon commented:
Thanks for the PR, but https://github.com/python/cpython is a cpython mirror repository; pull requests are not accepted on this repo. Please submit bugs and patches at https://bugs.python.org.

Mariatta committed:
bpo-29474: Improve documentation for weakref.WeakValueDictionary

There were some grammatical errors in weakref.WeakValueDictionary
documentation.
